app/components/statistic-list/component.js

```diff
@@ -71,7 +71,10 @@ export default class StatisticList extends Component {
     }
 
     const maxEstimated = moment.duration(
-      Math.max(0, ...this.value.mapBy("estimatedTime"))
+      Math.max(
+        0,
+        ...this.value.filterBy("estimatedTime").mapBy("estimatedTime")
```

Comment:
Why are you calling `filterBy` without a value?

Reply:
This filters out all `undefined` values. In case the list has only models without any entries with a `estimatedTime` attribute, it would return an empty array. `mapBy` on the other hand would return an array with `undefined` for each element.

```js
filterBy("estimatedTime") =>> []
mapBy("estimatedTime") =>> [undefined, undefined, ...] 
```

Reply:
Fair, as a nitpick if i get this correctly: `this.value.map(({estimatedTime})=>estimatedTime).filter(Boolean)` doesn't map the object by property twice. Or `this.value.mapBy("estimatedTime").filter(Boolean)` if you want to use `ArrayProxy` methods.

Reply:
:thinking: